pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -49,6 +56,22 @@ def upsample() -> nn.Upsample:
 
 
 class HourGlassBlock(SequentialWithOutChannels):
+    r"""HourGlassBlock from :cite:`ULVL2016`.
+
+    This block comprises a
+    :func:`~pystiche_paper.ulyanov_et_al_2016._modules.downsample`  to half the size
+    using :class:`~torch.nn.AvgPool2d` followed by a processing with ``intermediate``
+    and an :func:`~pystiche_paper.ulyanov_et_al_2016._modules.upsample` to twice the
```

Comment:
If the function is defined within the same module you can use relative links.
```suggestion
    :func:`downsample`  to half the size
    using :class:`~torch.nn.AvgPool2d` followed by a processing with ``intermediate``
    and an :func:`upsample` to twice the
```